lib/matplotlib/figure.py

```diff
@@ -1767,11 +1786,12 @@ def _normalize_grid_string(layout):
         else:
             # multi-line string
             layout = inspect.cleandoc(layout)
-            return [list(ln) for ln in layout.strip('\n').split('\n')]
+            return [list(_) for _ in layout.strip('\n').split('\n')]
```

Comment:
Why this change? `_` is usually used for *unused* values, which is not the case here.

Reply:
I don't remember why I did this, I'll put it back.